contracts/pcv/morpho/ICompound.sol

```diff
@@ -0,0 +1,398 @@
+// SPDX-License-Identifier: GNU AGPLv3
+pragma solidity ^0.8.0;
```

Comment:
Not for now, but in the vein of talking about how Morpho were impressive for locking down their npm package dependencies, might be worthwhile locking the Solidity version?

Reply:
agreed, solidity version in Morpho is locked to 0.8.13, this file should have been locked as well.